Makefile

```diff
@@ -40,15 +52,11 @@ elevate-cpanel: $(wildcard lib/**/*) script/elevate-cpanel.PL
 				       --leave-broken \
 				       script/$@.PL
 	mv script/$@.PL.static $@
-	MARKER="`cat maint/marker`" perl -pi -e 's|^(#!/usr/local/cpanel/3rdparty/bin/perl)|$$1\n\n$$ENV{MARKER}\n|' $@
-	VERSION=${VERSION} perl -pi -e 's/(^use constant VERSION =>) 1;/$$1 $$ENV{VERSION};/' $@
-	perltidy -b -bext="/" $@
+	MARKER="`cat maint/marker`" $(PERL_BIN)/perl -pi -e 's|^(#!/usr/local/cpanel/3rdparty/bin/perl)|$$1\n\n$$ENV{MARKER}\n|' $@
```

Comment:
I'm very unsure if this expands correctly inside the quotes???

Reply:
It expands like this:
```
MARKER="`cat maint/marker`" /usr/local/cpanel/3rdparty/perl/536/bin/perl -pi -e 's|^(#!/usr/local/cpanel/3rdparty/bin/perl)|$1\n\n$ENV{MARKER}\n|' elevate-cpanel
```